_layouts/post.liquid

```diff
@@ -18,9 +18,9 @@ layout: default
   <header class="post-header">
     <h1 class="post-title">{{ page.title }}</h1>
     <p class="post-meta">
-      Created: {{ page.date | date: '%B %d, %Y' }}
+      Created in {{ page.date | date: '%B %d, %Y' }}
       {% if page.author %}• {{ page.author }}{% endif %}
```

Comment:
Could you also change it in here for `by {{ page.author }}`? I think it will be more cohesive with the other changes.

Reply:
Sure! I have changed it.